src/Command/App.php

```diff
@@ -5,23 +5,33 @@
 namespace Codelicia\Xulieta\Command;
 
 use Codelicia\Xulieta\DocFinder;
-use Codelicia\Xulieta\Format\MarkdownDocumentationFormat;
 use Codelicia\Xulieta\Format\MultipleDocumentationFormat;
-use Codelicia\Xulieta\Format\RstDocumentationFormat;
 use InvalidArgumentException;
 use RuntimeException;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Finder\SplFileInfo;
+use function array_map;
 use function assert;
 use function sprintf;
 
 final class App extends Command
 {
     private bool $errorOccurred = false;
 
+    /** @var string[]  */
+    private array $config;
+
+    /** @param string[] $config */
```

Comment:
I'd be more precise here:

 * are these `array<int, string>`
 * are these `@psalm-var list<string>` maybe?
 * are the values of this list `@psalm-var non-empty-string`, resulting in `@psalm-var list<non-empty-string>`? Or `@psalm-var list<non-empty-string, string>`? Maybe the config could use a better-shaped VO?